lib/generators/boring/vcr/install/install_generator.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Boring
+  module Vcr
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds VCR to the application"
+      source_root File.expand_path('templates', __dir__)
+
+      class_option :test_framework,
+                   type: :string,
+                   alias: "-f",
+                   default: "rspec",
+                   enum: %w[rspec minitest],
+                   desc: "Tell us which test framework you are using. Default to rspec"
+
+      class_option :stubbing_libraries,
+                   type: :array,
+                   alias: "-sl",
+                   default: ['webmock'],
+                   enum: %w[webmock typhoeus faraday excon],
```

Comment:
I think it will be okay to support it as it does not need any configuration.

here is the text from documentation 
"Regardless of which library you use, VCR takes care of all of the configuration for you. You should not need to interact directly with WebMock or the stubbing facilities of Typhoeus, Excon or Faraday. If/when you decide to change stubbing libraries, you can change the hook_into configuration option and it'll work with no other changes required."
https://benoittgt.github.io/vcr/#/configuration/hook_into